src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -257,19 +277,7 @@ export const ArrayFieldTemplate: React.VFC<ArrayFieldTemplateProps> = ({
       {descriptionNode || (
         <DescriptionField id={`${id}__description`} description={description} />
       )}
-      {!readonly && canAdd && (
-        <ElementWrapper>
-          <Button
-            data-cy="add-button"
-            disabled={isDisabled}
-            leftGlyph={<Icon glyph="Plus" />}
-            onClick={onAddClick}
-            size={addButtonSize}
-          >
-            {addButtonText}
-          </Button>
-        </ElementWrapper>
-      )}
+      {!readonly && canAdd && !addToEnd && addButton}
```

Comment:
Is there  a way to make this conditional more clear on what its doing? e.g. turn addButton into a component and use JSX syntax so its more readable.

Reply:
why doesn't this need an AddButtonContainer?